<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1197" started-at="2014-12-15T10:56:43Z" finished-at="2014-12-15T10:56:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1197" started-at="2014-12-15T10:56:43Z" finished-at="2014-12-15T10:56:44Z">
      <class name="testCases.Framework_001">
        <test-method status="FAIL" signature="beforeMethod()[pri:0, instance:testCases.Framework_001@1d30b80]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2014-12-15T16:26:43Z" finished-at="2014-12-15T16:26:43Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[D:\ToolsQA\OnlineStore\src\testData\TestData.xlsx (The device is not ready)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: D:\ToolsQA\OnlineStore\src\testData\TestData.xlsx (The device is not ready)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at utility.ExcelUtils.setExcelFile(ExcelUtils.java:18)
	at testCases.Framework_001.beforeMethod(Framework_001.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="main()[pri:0, instance:testCases.Framework_001@1d30b80]" name="main" duration-ms="2" started-at="2014-12-15T16:26:44Z" finished-at="2014-12-15T16:26:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main -->
        <test-method status="SKIP" signature="afterMethod()[pri:0, instance:testCases.Framework_001@1d30b80]" name="afterMethod" is-config="true" duration-ms="0" started-at="2014-12-15T16:26:44Z" finished-at="2014-12-15T16:26:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testCases.Framework_001 -->
      <class name="testCases.Framework_002">
        <test-method status="FAIL" signature="beforeMethod()[pri:0, instance:testCases.Framework_002@1060c3e]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2014-12-15T16:26:44Z" finished-at="2014-12-15T16:26:44Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[D:\ToolsQA\OnlineStore\src\testData\TestData.xlsx (The device is not ready)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: D:\ToolsQA\OnlineStore\src\testData\TestData.xlsx (The device is not ready)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at utility.ExcelUtils.setExcelFile(ExcelUtils.java:18)
	at testCases.Framework_002.beforeMethod(Framework_002.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="f()[pri:0, instance:testCases.Framework_002@1060c3e]" name="f" duration-ms="0" started-at="2014-12-15T16:26:44Z" finished-at="2014-12-15T16:26:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="SKIP" signature="afterMethod()[pri:0, instance:testCases.Framework_002@1060c3e]" name="afterMethod" is-config="true" duration-ms="0" started-at="2014-12-15T16:26:44Z" finished-at="2014-12-15T16:26:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testCases.Framework_002 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
